containers/Ideas/IdeaCreateDescription.js

```diff
@@ -0,0 +1,146 @@
+import React, { useState } from 'react';
+import { Alert, View } from 'react-native';
+import { Button } from 'react-native-elements';
+import { Formik } from 'formik';
+import * as yup from 'yup';
+import AsyncStorage from '@react-native-async-storage/async-storage';
+import IconSLI from 'react-native-vector-icons/SimpleLineIcons';
+import { ButtonSubmit } from '../../components/ButtonSubmit';
+import { TextSourceSans } from '../../components/TextSourceSans';
+
+import { styles } from './Idea.styles';
+import { TextInputFullFormField } from '../../components/formFields';
+import API from '../../BaseApi';
+
+export const IdeaCreateDescription = props => {
+
+  const {idea, project, editing} = props.route.params;
+  const moduleId = project.single_agenda_setting_module;
+  const arrowLeftIcon = (
+    <IconSLI name='arrow-left' size={22} />
+  );
+  const ideaDescriptionValidationSchema = yup.object().shape({
+    description: yup
+      .string()
+      .max(10000, 'Description must be no longer then 100 characters')
+      .required('Description is required'),
+  });
+
+  const [error, setError] = useState();
+
+  const handleSubmit = (values) => {
```

Comment:
do we want to actual submit the description text separately from the IdeaCreate screen?
Or should we rather send it back to IdeaCreate and submit it to the backend together with the other fields?

Reply:
I don't know, I kind of thought it would have to be submitted  for some reason but I guess the textinput value can just be sent back to the form in props and submitted as part of the IdeaCreate form right?

Reply:
okay, yes i know this is part2 of the work. But so you also prefer to send it back and rather send it together with the rest right? Then leave it like that and we will see once we work on part2.

Reply:
yes I think that would be best as the title of the toDescription button will need to change when description is added so the form will need to know it